lib/workers/pr/changelog/release-notes.ts

```diff
@@ -67,16 +74,16 @@ function massageBody(input, githubBaseURL) {
   return body.trim();
 }
 
-async function getReleaseNotes(
-  repository,
-  version,
-  depName,
-  githubBaseURL,
-  githubApiBaseURL
-) {
+export async function getReleaseNotes(
+  repository: string,
+  version: string,
+  depName: string,
+  githubBaseURL: string,
+  githubApiBaseURL: string
+): Promise<ChangeLogNotes | undefined> {
```

Comment:
`null`?

Reply:
It's because the result object can be never set. I'll change it to be explicit null.

Reply:
https://github.com/renovatebot/renovate/blob/cd865d1e25f821885641acfd51af44a31c7e9017/lib/workers/pr/changelog/release-notes.ts#L97